Tests/MMEMetricsManagerTests.mm

```diff
@@ -270,6 +270,36 @@ + (NSString *)pendingMetricsEventPath;
                 [manager loadPendingTelemetryMetricsEvent] should be_nil;
             });
         });
+        context(@"when incrementing location events related counters", ^{
```

Comment:
Can we move these to XCtests? It's OK if there are some tests in cedar still but I don't want to add any more cedar tests to the project.

We also need tests to cover the persistence of these values, this is the only event we write to storage and restore so we should double check